### PR TITLE
Paginate customer table

### PR DESCRIPTION
**Pull request by fmterrorf**
### Description
Add pagination for the customer table in the following pages
* Customers > People
* Customers > Companies > Select a company 
* Customers > Tags > Select a tag

### Issue
#708 

### Screenshots

* Customers > People

https://user-images.githubusercontent.com/77761194/114246290-7b001080-994f-11eb-8fbc-bd222c5afdc8.mp4

* Customers > Companies > Select a company 

https://user-images.githubusercontent.com/77761194/114246307-87846900-994f-11eb-9e99-398240c3d40d.mp4

* Customers > Tags > Select a tag

https://user-images.githubusercontent.com/77761194/114246316-8d7a4a00-994f-11eb-96f4-6a08ffdb55f7.mp4

## Checklist

- [x] Everything passes when running `mix test`
- [x] Ran `mix format`
- [x] No frontend compilation warnings

### CODE REVIEW

File: config/dev.exs [hunk 1]
```diff
@@ -6,8 +6,7 @@ database_url = System.get_env("DATABASE_URL") || "ecto://postgres:postgres@local
 config :chat_api, ChatApi.Repo,
   url: database_url,
   show_sensitive_data_on_connection_error: true,
-  pool_size: 10,
-  socket_options: [:inet6]
+  pool_size: 10
```
Comment (reichert621) [inline config/dev.exs:9]: just curious, why this change? is it necessary for this PR?

Comment (fmterrorf) [reply config/dev.exs:9]: shoot sorry. I'll revert this. I use it when running the project locally

File: assets/src/components/customers/CustomersTableContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+  loading: boolean;
+  query: string;
+  customers: Array<Customer>;
+  filteredCustomers: Array<Customer>;
+  pagination: Pagination;
+};
+
+const DEFAULT_PAGE_SIZE = 10;
+
+class CustomersTableContainer extends React.Component<Props, State> {
```
Comment (fmterrorf) [inline assets/src/components/customers/CustomersTableContainer.tsx:27]: Thanks for separating to another component!

Comment (reichert621) [reply assets/src/components/customers/CustomersTableContainer.tsx:27]: no prob, just wanted to remove a bunch of the duplicate code :) :+1:
